.ci/magician/cmd/generate_comment.go

```diff
@@ -154,6 +154,12 @@ func listGCEnvironmentVariables() string {
 func execGenerateComment(prNumber int, ghTokenMagicModules, buildId, buildStep, projectId, commitSha string, gh GithubClient, rnr ExecRunner, ctlr *source.Controller) error {
 	errors := map[string][]string{"Other": []string{}}
 
+	// TODO(ScottSuarez) - temporary fix to ensure the label is removed.
+	// Once we migrate to the new trigger there is an explicit task
+	// for this and this line can be removed.
+	gh.RemoveLabel(fmt.Sprint(prNumber), "awaiting-approval")
```

Comment:
Since you're adding the new step to the gcb-generate-diffs-new.yml do we still need this?

Reply:
Yes because the YAML on contributors PR won't see the new change until that customer pulls the latest changes. Which means the step won't be present for old PRs.

We can remove after two weeks.